source/guides/community-chat.rst

```diff
@@ -1,26 +1,44 @@
-Mattermost community collaboration server
-=========================================
+Join our community
+==================
 
-Mattermost hosts a collaboration and chat server at `community.mattermost.com <https://community.mattermost.com/login>`__ to connect our team with our larger contributor and user community. This server is open to everyone, and we welcome anyone who wants to participate in our community to sign up.
+Connect with thousands of contributors, customers, and users to build, share, and learn together at `community.mattermost.com <https://community.mattermost.com>`__. This server is our virtual office and is open to everyone. Please review our `Code of Conduct <https://handbook.mattermost.com/contributors/contributors/guidelines/contribution-guidelines>`__ before participating.
 
-We have many channels on our community server you can join. Here are some of the more popular channels.
+General channels
+----------------
 
-**General purpose channels**
+- `~Contributors <https://community.mattermost.com/core/chann0els/tickets>`__
+- `~Community Team <https://community.mattermost.com/core/channels/community-team>`__
```

Comment:
Were we considering to remove this channel in favor of the peer-to-peer help channel? https://community.mattermost.com/core/channels/peer-to-peer-help

Reply:
Which channel are you referring to here? ~Community Team? We want to keep this channel and use it moving forward as a focused sharing spot for community projects, events, and communication. This channel is a good candidate for engaging the community because it's currently pretty bare but well-populated.

Reply:
Sorry, was referring to the Community Team. I thought we were looking to deprecate it. I'm 0/5, we can keep it for now if high impact!